spec/mailers/member_mailer_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe MemberMailer, :type => :mailer do
+  describe "send_message" do
+    let(:member) { create(:member, email: 'example@mail.com')}
+    let(:user){ create(:user)}
+    let(:mail) { MemberMailer.send_message( 'subject1', 'body one', member.id, user.id) }
+
+    it "renders the headers" do
+      expect(mail.subject).to eq("subject1")
+      expect(mail.to).to eq(["example@mail.com"])
+    end
+
+    it "renders the body" do
+      expect(mail.body.encoded).to match("body one")
+    end
+
+    it "job is created" do
```

Comment:
recuerda que los `it` deben quedar como una frase leíble en inglés. Algo como `it "enqueues the mail" `